linode/helper/framework_data.go

```diff
@@ -17,6 +17,25 @@ func KeepOrUpdateBool(original types.Bool, updated bool, preserveKnown bool) typ
 	return KeepOrUpdateValue(original, types.BoolValue(updated), preserveKnown)
 }
 
+func KeepOrUpdateStringPointer(original types.String, updated *string, preserveKnown bool) types.String {
+	return KeepOrUpdateValue(original, types.StringPointerValue(updated), preserveKnown)
+}
+
+func KeepOrUpdateInt64Pointer(original types.Int64, updated *int64, preserveKnown bool) types.Int64 {
+	return KeepOrUpdateValue(original, types.Int64PointerValue(updated), preserveKnown)
+}
+
+func KeepOrUpdateIntPointer(original types.Int64, updated *int, preserveKnown bool) types.Int64 {
+	if updated == nil {
+		return KeepOrUpdateValue(original, types.Int64Null(), preserveKnown)
```

Comment:
nitpick: Just for clarity, could we have a comment here explaining that ints do not have a corresponding `types.XPointerValue`?